app/translations/en.json

```
@@ -369,6 +369,14 @@
     },
     "widgets": {
         "common": {
+            "labels": {
+                "keyPlaceholder": "Select or type in a key",
+                "newKey": "New key",
+                "newValue": "New value",
+                "valuePlaceholder": "Select or type in a value",
+                "validationError": "Only letters, digits and the characters \"-\", \".\" and \"_\" are allowed and it must begin with a letter.",
+                "isLabelInSystemCheckError": "Cannot check is label already present in the system",
```

A trailing comma here is invalid JSON :worried: 

```suggestion
                "isLabelInSystemCheckError": "Cannot check is label already present in the system"
```